pkg/interface/src/logic/state/metadata.ts

```diff
@@ -51,7 +51,7 @@ const useMetadataState = createState<MetadataState>('Metadata', {
   //     });
   //   });
   // },
-});
+}, [], 2);
```

Comment:
A lot of potential deploy overhead on this approach! Having to increment the version each time we edit. Why can’t we just inject something each build? On livenet this is rare, on devnet it will just mean inconsistent time to paint.